src/components/dialogs/network-modifications/tabular-modification/tabular-modification-dialog.js

```diff
@@ -72,6 +72,16 @@ const TabularModificationDialog = ({
 
     const { reset } = formMethods;
 
+    const getValueFromEnum = useCallback(
```

Comment:
You can use the same method as https://github.com/gridsuite/gridstudy-app/pull/1773 : 
const getEnumIdFromLabel = useCallback(
        (label, enumValues) =>
            enumValues.find(
                (es) => intl.formatMessage({ id: es.label }) === label
            )?.id,
        [intl]
    );